appcenter-auth/Auth.js

```diff
@@ -6,18 +6,45 @@ const ReactNative = require('react-native');
 const { AppCenterReactNativeAuth } = ReactNative.NativeModules;
 
 const Auth = {
-    isEnabled() {
-        return AppCenterReactNativeAuth.isEnabled();
-    },
-    setEnabled(enabled) {
-        return AppCenterReactNativeAuth.setEnabled(enabled);
-    },
-    signIn() {
-        return AppCenterReactNativeAuth.signIn();
-    },
-    signOut() {
-        AppCenterReactNativeAuth.signOut();
-    }
+    isEnabled,
+    setEnabled,
+    signIn,
+    signOut
 };
 
+/**
+ * Check whether Auth service is enabled or not.
+ *
+ * @return Future asynchronous operation with result being true if enabled, false otherwise.
+ */
+function isEnabled() {
+    return AppCenterReactNativeAuth.isEnabled();
+}
+
+/**
+ * Enable or disable Auth service.
+ *
+ * @param {boolean} enabled - true to enable, false to disable.
+ * @return Future asynchronous operation with null result to monitor when the operation completes.
+ */
+function setEnabled(enabled) {
+    return AppCenterReactNativeAuth.setEnabled(enabled);
+}
+
+/**
+ * Sign in to get user information.
+ *
+ * @return Future asynchronous operation with the result of the sign-in operation.
```

Comment:
future is a java terminology, Javascript uses promises

Reply:
fixed